Userland/Libraries/LibVideo/VP9/MV.h

```diff
@@ -13,6 +13,7 @@ namespace Video::VP9 {
 class MV {
 public:
     MV() = default;
+    MV(MV const& other);
```

Comment:
this and operator=(MV const&) should be `= default;` in the header here. As well as defaulting a move constructor and move assign. Rule of 5 and all that.

Reply:
I'll just remove that line, since I'm not sure how that snuck in there considering it's not necessary.

Reply:
With regard to the rule of 5, I'll add the other functions if needed in my constexpr refactor of MotionVector.